mlocal/frags/Makefile.stub

```diff
@@ -67,7 +67,7 @@ $(wrapper_OBJ): $(libstartup) $(SOURCEDIR)/src/runtime/startup/c/wrapper.c
 $(wrapper): $(wrapper_OBJ) $(libstartup)
 	@echo " LD" $@
 	@mkdir -p $(@D)
-	$(V)$(CC) $(LDFLAGS) -o $@ $(wrapper_OBJ) -Wl,--start-group -L$(BUILDDIR)/lib -lruntime -lstartup -Wl,--end-group -luuid -lpthread $(WRAPPER_LIBS)
+	$(V)$(CC) $(LDFLAGS) -o $@ $(wrapper_OBJ) -Wl,--start-group -L$(BUILDDIR)/lib -lruntime -lstartup -Wl,--end-group -luuid -lpthread -lgpgme $(WRAPPER_LIBS)
```

Comment:
What is the gpgme for ? Do we import a package that requires it ?

Reply:
Yes, this is an unfortunate side effect of using `build.Definition` as a member field of `src/runtime/engines/imgbuild/config.EngineConfig`. `build` package needs `-lgpgme`. Maybe this means the `Definition` type should be moved to a buildtypes subpackage?